releases/latest/kernel/helpers/build/configure.sh

```diff
@@ -77,5 +77,5 @@ then
   fi
 fi
 
-# Server start/stop to populate the /output/workarea and make subsequent server starts faster
-/opt/ol/wlp/bin/server start && /opt/ol/wlp/bin/server stop && rm -rf /output/messaging /logs/* $WLP_OUTPUT_DIR/.classCache && chmod -R g+rwx /opt/ol/wlp/output/*
+# Create a new SCC layer, args are layer size and number of iterations to run to populate it
+populate_scc.sh 8m 3
```

Comment:
what happens if a cache larger than 8m is needed?

Reply:
If you mean what happens at runtime if we fill the cache to capacity then nothing, it's not an error, the program continues to run without any issue and the JVM stops trying to cache any additional data.

If you mean what happens if someone wants to specify a larger cache, they can't at the moment. We can add an argument to the `configure.sh` script that can then be passed down to `populate_scc.sh` if you want. That would allow dependent images to have some way to specify the size of their SCC layer.

Reply:
Right - I was wondering about the first case.    Thx.